.travis.yml

```diff
@@ -0,0 +1,35 @@
+language: minimal
+
+branches:
+  only:
+    - master
+    - nightly-tests
+
+before_install:
+  # Decrypt secrets
+  - openssl aes-256-cbc -K $encrypted_36785d287aa1_key -iv $encrypted_36785d287aa1_iv
+    -in ci/keys/substra-208412-3be0df12d87a.json.enc -out ci/keys/substra-208412-3be0df12d87a.json
+    -d
+  - openssl aes-256-cbc -K $encrypted_3657ea9362f7_key -iv $encrypted_3657ea9362f7_iv
+    -in ci/keys/kaniko-secret.json.enc -out ci/keys/kaniko-secret.json -d
+  # Install helm (v2.14.2)
+  - curl https://get.helm.sh/helm-v2.14.2-linux-amd64.tar.gz -o helm-v2.14.2-linux-amd64.tar.gz
+  - tar xzf helm-v2.14.2-linux-amd64.tar.gz
+  - mv linux-amd64/helm linux-amd64/tiller $HOME/bin/
+  # Install kubectl
+  - curl -LO https://storage.googleapis.com/kubernetes-release/release/`curl -s https://storage.googleapis.com/kubernetes-release/release/stable.txt`/bin/linux/amd64/kubectl
+  - chmod +x ./kubectl
+  - mv ./kubectl $HOME/bin/
```

Comment:
@ClementGautier Is there a way to install package in travis or maybe a image exist to avoid installing helm and kubectl manually :) ?

Reply:
It's add addons :) 
https://docs.travis-ci.com/user/job-lifecycle/#the-job-lifecycle

Reply:
If I understand well, travis `addon` is just a syntax, it will execute nearly the same command (apt, snap, brew, etc.)
It's just a travis way to install dependencies. Nevertheless, I don't know if it's better or not to use it ! 
@ClementGautier  Do you have some insights to share on it ?

Reply:
Suing addons may be more robust as it uses a miror on travis side to cache packages. It's also easier to read in my opinion but that's a matter of taste.

Reply:
(I should have mentioned it before) I initially used addons but snap only has helm 2.15 (not 2.14). I couldn't get things to work with 2.15. We could give it another shot though.